.gitignore

```diff
@@ -2,3 +2,5 @@ coverage.txt
 bin
 card.png
 dist
+
+!.editorconfig
```

Comment:
What is this syntax about?

Overload a possible global gitignore that may exist?

I would say this line shouldn't be there now you added .editorconfig via a commit. The file is now tracked.

Please note I might be wrong

Reply:
> Overload a possible global gitignore that may exist?

Yes

Reply:
However, I have already added the file to the index - the absence of this rule will not affect the result. I'll remove it